proj/hps_accel/Makefile

```diff
@@ -98,12 +104,19 @@ ifeq '$(TARGET)' 'digilent_arty'
 EXTRA_LITEX_ARGS += --sys-clk-freq 75000000
 endif
 
+ifneq '$(PARALLEL)' 'no'
+EXTRA_LITEX_ARGS += --extra-nextpnr-params
+EXTRA_NEXTPNR_ARGS += --placer-heap-timingweight 79
+endif
```

Comment:
I feel these two things (parallel build & parameter adjustment) shouldn't necessarily be tied together.

Reply:
Actually it's fine, we can adjust later as necessary.  However, is router1 used by default?   Do the params make sense with router1?

Unfortunately I have let it get quite complicated, between the options in Makefile and various scripts (and CI flows)...

Reply:
I have removed this, as it is actually not in the scope of allowing parallel sw/hw builds